nekoyume/Assets/_Scripts/UI/Widget/Tooltip/SimpleTooltip.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using Nekoyume.Model.Item;
+using TMPro;
+using UnityEngine;
+
+namespace Nekoyume.UI.Module
+{
+    public class SimpleTooltip : Widget
+    {
+        [SerializeField] private TextMeshProUGUI tooltipText;
+
+        private RectTransform _rectTransform;
+        private Coroutine _waitCoroutine;
+
+        protected override void Awake()
+        {
+            base.Awake();
+            _rectTransform = GetComponent<RectTransform>();
+            gameObject.SetActive(false);
+        }
+
+        public void Show(ItemBase itemBase, Transform target)
+        {
+            Show(itemBase.GetLocalizedDescription(), target);
+        }
+
+        public void Show(string message, Transform target)
+        {
+            tooltipText.text = message;
+            _rectTransform.position = target.position;
+            gameObject.SetActive(true);
+            StartCoroutine(CoWaitClose());
+            base.Show();
+        }
+
+        private IEnumerator CoWaitClose()
+        {
+            while (gameObject.activeSelf)
```

Comment:
If the gameobject of Coroutine is disabled, all unity method will be turned off, which is unnecessary.